code/dispatcher/dispatcher.php

```diff
@@ -81,23 +78,23 @@ public static function getInstance(KObjectConfigInterface $config, KObjectManage
     }
 
     /**
-     * Load the controller translations
+     * Resolve the request
      *
-     * @param KControllerContextInterface $context
-     * @return void
+     * @param KDispatcherContextInterface $context A dispatcher context object
+     * @throw KDispatcherExceptionMethodNotAllowed If the HTTP request method is not allowed.
      */
-    protected function _loadTranslations(KControllerContextInterface $context)
+    protected function _resolveRequest(KDispatcherContextInterface $context)
     {
-        $package = $this->getIdentifier()->package;
-        $domain  = $this->getIdentifier()->domain;
+        parent::_resolveRequest($context);
 
-        if($domain) {
-            $identifier = 'com://'.$domain.'/'.$package;
-        } else {
-            $identifier = 'com:'.$package;
+        //Resolve the controller action
+        $method = strtolower($context->request->getMethod());
+
+        if (!in_array($method, $this->getHttpMethods())) {
+            throw new KDispatcherExceptionMethodNotAllowed('Method not allowed');
         }
 
-        $this->getObject('translator')->load($identifier);
+        $this->setControllerAction($method);
```

Comment:
This effectively removes support for _action override though which is used in a lot of places for sending delete, move, copy requests for example.